lib/scrub/scrub_runner.rb

```diff
@@ -98,11 +99,11 @@ def run_file(file)
           Loader::HoldingLoader::Cleanup.new.on_success(:success, cleanup_data)
         end
       end
-    rescue => err
+    rescue MalformedFileError => err
```

Comment:
Took me a minute to understand this -- now we are catching specifically `MalformedFileError` rather than logging & re-raising an error here. But, we are also changing the behavior not to catch unexpected errors & remove the downloaded file -- is that OK?

Reply:
Yeah you are right... this `rescue` _does_ do the job but it is too specific. Changing it.